samples/mqtt_example/test/cloudiot_mqtt_example.test.js

```diff
@@ -32,11 +32,12 @@ const registryName = `nodejs-iot-test-mqtt-registry-${uuid.v4()}`;
 const region = 'us-central1';
 const rsaPublicCert = 'resources/rsa_cert.pem'; //process.env.NODEJS_IOT_RSA_PUBLIC_CERT;
 const rsaPrivateKey = 'resources/rsa_private.pem'; //process.env.NODEJS_IOT_RSA_PRIVATE_KEY;
+const serverCert = 'resources/roots.pem'; //process.env.NODEJS_IOT_SERVER_CERT;'
```

Comment:
Be sure to run`prettier` on this file. I _believe_ that the correct style is to put code comments above the line they're commenting.

Reply:
According to prettier, this style is correct.